project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clanhr/analytics "1.9.1"
+(defproject clanhr/analytics "1.9.2"
```

Comment:
actually this is a bugfix on the project side and not on clanger/analytics side. I'll change that 👍